spark/pom.xml

```diff
@@ -42,6 +42,8 @@
     <hadoop.version>2.3.0</hadoop.version>
     <yarn.version>${hadoop.version}</yarn.version>
     <jets3t.version>0.9.3</jets3t.version>
+
+    <spark.download.url>http://www.apache.org/dist/spark/spark-${spark.version}/spark-${spark.version}.tgz</spark.download.url>
```

Comment:
*clean build* from my previous message means CI on Travis virtual machine as well as just installed development environment.

So, are you you sure it will not download it in those cases?